wrapper-specification.md

```diff
@@ -68,7 +68,7 @@ Position P|int|Read/Write| The proportional constant for the position PID.
 Position I|int|Read/Write| The integral constant for the position PID.
 Position D|int|Read/Write| The derivative constant for the position PID.
 Position SP|int|Read/Write| Writing specifies the target position for the `run-to-abs-pos` and `run-to-rel-pos` commands. Reading returns the current value. Units are in tacho counts. You can use the value returned by `counts_per_rot` to convert tacho counts to/from rotations or degrees.
-Speed|int|Read| Returns the current motor speed in tacho counts per second. Not, this is not necessarily degrees (although it is for LEGO motors). Use the `count_per_rot` attribute to convert this value to RPM or deg/sec.
+Speed|int|Read| Returns the current motor speed in tacho counts per second. Note, this is not necessarily degrees (although it is for LEGO motors). Use the `count_per_rot` attribute to convert this value to RPM or deg/sec.
```

Comment:
No, I am wrong. This *is* an autogenerated file, but we still want to put the change here.